deploy/chart/templates/0000_50_18-service-monitor.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: olm-operator
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: olm-operator
+spec:
+  jobLabel: k8s-app
+  endpoints:
+  - port: http-metrics
+    interval: 30s
+    scheme: http
+    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
Sending bearer tokens un-encrypted over the wire sounds a bit scary to me. Given that this would be the Prometheus service account token, one could gain quite a bit, impersonating it. What do you think?

Reply:
If this isn't required for http scraping, then I'll certainly remove it.

Reply:
@jpeeler whether it is necessary or not depends on how you secure your metrics endpoint on the operator side.

What I was referring to with my comment above was, that you are using `scheme: http`, thereby sending the bearer token in the line below in clear text over the wire. Given that one can impersonate Prometheus once they captures the token, this seems insecure to me.

Reply:
I've switched everything over to https.